src/apps/Permissions/EntitySelector.js

```diff
@@ -46,7 +52,7 @@ class EntitySelector extends React.Component {
   }
   getAddress(index) {
     if (index === -1 || index === 0) {
-      return getEmptyAddress()
+      return ''
```

Comment:
I modified this call so that it returns an empty string if it's set to "Assign Entity" (the initial state). This was because `getEmptyAddress()` was triggering the "unassigned" warning message, and this is not the intended behavior, I assume.